composer.json

```diff
@@ -18,7 +18,7 @@
     ],
     "require": {
         "php" : ">=7.0",
-        "yubico/u2flib-server": "1.0.2",
+        "yubico/u2flib-server": "1.0.1",
```

Comment:
It you PR https://github.com/Yubico/php-u2flib-server/pull/75 can you re-update the package ?

Reply:
Yes, I hope the fix will be merged, they consider the library being deprecated in favour of Webauthn but there is no package ready for prime time yet...